app/src/main/java/com/poupa/vinylmusicplayer/ui/fragments/player/card/CardPlayerPlaybackControlsFragment.java

```diff
@@ -49,6 +51,10 @@
 
     private MusicProgressViewUpdateHelper progressViewUpdateHelper;
 
+    private final int INITIAL_INTERVAL_MILLI = 1000;
```

Comment:
Is that possible to move these defaults to the previous class, i.e. keep them close to where they make sense.

Or rename them to clarify the usage (i,e. SKIP_TRIGGER_INTERVAL_MILLIS)

Reply:
Resolved in the most recent commit